kalite/control_panel/static/js/control_panel/bundle_modules/data_export.js

```diff
@@ -0,0 +1,9 @@
+var $ = require("base/jQuery");
+var DataExportView = require("data_export/views").DataExportView;
+var DataExportStateModel = require("data_export/models").DataExportStateModel;
+
+module.exports = {
+	$: $,
```

Comment:
How does this work? You're exporting JQuery yourself, and so anyone who imports `data_export.js` automatically imports jquery into their namespace too?